os/include/net/lwip/opt.h

```diff
@@ -59,6 +59,7 @@
  * will be set to standard values. Override anything you don't like!
  */
 #include <net/lwip/lwipopts.h>
+#include <net/lwip/debug.h>
```

Comment:
opt.h should not refer lwip/debug.h because it could create circular dependency. for example LWIP_DEBUGF

Reply:
@pillip8282 : debug.h is not required. But if we dont include this travis-c  is failing.

Reply:
@vinaykadam1  you should not include debug.h at opt.h because already debug.h refers opt.h. you can fix travis-c failing by another way.